docs/usage/configuration-options.md

```diff
@@ -1374,6 +1374,19 @@ For example to apply a special label for Major updates:
 }
 ```
 
+### maxBranchLength
```

Comment:
I know I suggested this name originally, but we maybe need a different one. Especially because configuring it results in a *fixed* branch length, not a maximum one. What do you think about `hashedBranchLength`?